pylint/checkers/misc.py

```diff
@@ -133,16 +148,29 @@ def process_tokens(self, tokens: list[tokenize.TokenInfo]) -> None:
         if not self.linter.config.notes:
             return
         for token_info in tokens:
-            if token_info.type != tokenize.COMMENT:
-                continue
-            comment_text = token_info.string[1:].lstrip()  # trim '#' and white-spaces
-            if self._fixme_pattern.search("#" + comment_text.lower()):
-                self.add_message(
-                    "fixme",
-                    col_offset=token_info.start[1] + 1,
-                    args=comment_text,
-                    line=token_info.start[0],
-                )
+            if token_info.type == tokenize.COMMENT:
+                comment_text = token_info.string[1:].lstrip()  # trim '#' and white-spaces
+                if self._comment_fixme_pattern.search("#" + comment_text.lower()):
+                    self.add_message(
+                        "fixme",
+                        col_offset=token_info.start[1] + 1,
+                        args=comment_text,
+                        line=token_info.start[0],
+                    )
+            elif self.linter.config.check_fixme_in_docstring and self._is_docstring_comment(token_info):
+                docstring_lines = token_info.string.split("\n")
+                for line_no, line in enumerate(docstring_lines):
+                    comment_text = line.removeprefix('"""').lstrip().removesuffix('"""')  # trim '""""' and whitespace
```

Comment:
`removeprefix()` and `removesuffix()` are new in python 3.9. Dumb question, but how do I tell what version this project supports?

Reply:
After seeing 3.8 test suites fail, I'm guessing I need to make this compatible 😅

Reply:
Do we really need this? Can't we just put the full docstring in the message for now?

Reply:
For a case like 
```
"""
TODO msg1
TODO msg2
"""
```
this PR will create two TODO lint messages. If we put the full docstring for both, it might be confusing or overly wordy, no?